application-analytics-ui/src/main/resources/Analytics/Code/Macros/VisitsSummary.xml

```diff
@@ -250,6 +250,17 @@ require(['jquery', 'xwiki-l10n!analyticsGraphMessages', 'chart'], function ($, l
         chart.destroy();
       }
       let values = fields.map((field, i) =&gt; responses[i]['values']);
+      if(responses[0]["labels"].length === 1)
```

Comment:
I'd add a comment explaining why it's needed to add the additional points. Otherwise looks good.